app/src/main/res/layout/item_help_faq_category.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <import type="cz.covid19cz.erouska.R" />
+
+        <variable
+            name="item"
+            type="cz.covid19cz.erouska.ui.help.data.FaqCategory" />
+
+        <variable
+            name="vm"
+            type="cz.covid19cz.erouska.ui.help.HelpVM" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:onClick="@{ () -> vm.onItemClicked(item)}"
+        android:paddingVertical="6dp">
+
+        <ImageView
+            android:id="@+id/category_icon"
+            android:layout_width="22dp"
+            android:layout_height="22dp"
+            android:layout_marginStart="17dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:placeholder="@{R.drawable.ic_item_empty}"
+            app:url="@{item.icon}"
```

Comment:
Can we try to use local vector SVGs as there won't be that many categories?

Reply:
the urls will be provided in the RC and since the help RC may change and it is all dynamic, it's nearly impossible to map the categories to local drawables